### PR TITLE
h22 done

### PR DESCRIPTION
**Pull request by andrjuxa201185**
@iliatcymbal PTAL

### CODE REVIEW

File: src/pages/userLogin/Userlogin.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import { Notfound } from '../notfound';
+import { Usercabinet } from '../usercabinet';
+
+
+export const Userlogin = () => (
+  <Switch>
+    <Route
+      path="/"
+      exact
+      component={() => <Usercabinet />}
```
Comment (iliatcymbal) [inline src/pages/userLogin/Userlogin.js]: why don't `component={Usercabinet}`?

File: src/pages/Pages.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// import { Route, Switch, Redirect } from 'react-router-dom';
+
+import { Userlogin } from './userlogin';
```
Comment (iliatcymbal) [inline src/pages/Pages.js]: you don't have such folder `userlogin`

File: src/components/loader/index.js [hunk 3]
```diff
@@ -0,0 +1 @@
+export * from './Loader';
```
Comment (iliatcymbal) [inline src/components/loader/index.js]: you don't have file with such name

**Comment by andrjuxa201185** [comment]
@iliatcymbal PTAL

**Comment by iliatcymbal** [comment]
What about [this one](https://github.com/andrjuxa201185/andrey_podgorniy_shop/pull/15#discussion_r284922098)?

**Comment by andrjuxa201185** [comment]
@iliatcymbal, I have renamed file login.js to Login.js.

**Comment by andrjuxa201185** [comment]
I will never use again "mrec" )))

**Comment by iliatcymbal** [comment]
Still I don't see the changes
https://github.com/andrjuxa201185/andrey_podgorniy_shop/tree/hw22/src/components/loader
http://i.imgur.com/fA0DZKU.png

**Comment by andrjuxa201185** [comment]
@iliatcymbal. GIT didn't see the case difference and did not push (
Now there is OK

**Comment by iliatcymbal** [comment]
You can use `git config core.ignorecase false`

**Pull request merged**